docs/developers/add-new-analytics-module.md

```diff
@@ -0,0 +1,19 @@
+# Analytics module 
+
+This document describes how to add a new analytics module to Prebid Server. 
+
+The PBS Analytics creates _loggable objects_ for a transaction at each endpoint (for example, an `AuctionObject` for the `/openrtb2/auction` endpoint, `CookieSyncObject` for the  `/cookiesync` endpoint, `SetUIDObject` for the `/setuid` endpoint, and an `AmpObject` for the `/amp` endpoint.)
+
+## Steps to add new module
+
+### In analytics package,
+
+1. Create a file for the new module.
+2. Have the new module implement `PBSAnalyticsModule` interface that allows it to extract and log necessary information from the loggable objects. 
```

Comment:
You can make this a link to the source code. Surround text in `[]`, and put the URL in `()`. See any of the other `.md` files for an example.